extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -718,6 +719,25 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
         Services.documentInfoForDoc(this.element).canonicalUrl,
     };
   }
+  
+  /**
+   * Returns the referrer or undefined if the referrer is not resolved
+   * before the given timeout
+   * @param {number=} opt_timeout
+   * @return {!Promise<string|undefined>} A promise for a referrer or undefined 
+   * @private
+   */
+  getReferrer_(opt_timeout) {
+    const timeoutInt = parseInt(opt_timeout, 10);
+    const refererPromise = Services.viewerForDoc(this.getAmpDoc())
+      .getReferrerUrl();
+    if (opt_timeout == null || isNaN(opt_timeout) || opt_timeout < 0) {
```

Comment:
@keithwrightbos I added `null` also here (consistency with `ADCID`

Reply:
Isn't necessary as parseInt(null, 10) and parseInt(undefined, 10) both return NaN